app/models/importers/content_items_by_organisation.rb

```diff
@@ -11,9 +11,14 @@ def run(slug)
 
     def create_or_update!(attributes, organisation)
       content_id = attributes.fetch(:content_id)
-      content_item = organisation.content_items.find_or_create_by(content_id: content_id)
+      content_item = ::ContentItem.find_or_create_by(content_id: content_id)
 
-      content_item.update!(attributes)
+      content_item.update!(attributes.except(:content_id))
+
+      organisations = content_item.organisations
+      unless organisations.include?(organisation)
+        organisations << organisation
+      end
```

Comment:
Very minor: This reads to me a bit better. The `unless` clause is not crucial to understand what the method does. Inlining it feels ok.

```ruby
    def create_or_update!(attributes, organisation)
      content_id = attributes.fetch(:content_id)
      content_item = ContentItem.find_or_create_by(content_id: content_id)

      content_item.update!(attributes)
      content_item.organisations << organisation unless content_item.organisations.include?(organisation)
    end
```